Projects/Feature/History/Sources/Component/CheckBoxEditTitleBottomSheetView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CheckBoxEditTitleBottomSheetView.swift
+//  FeatureHistory
+//
+//  Created by 현수빈 on 11/26/24.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+import SharedDesignSystem
+
+struct CheckBoxEditTitleBottomSheetView: View {
+  @Bindable var store: StoreOf<HistoryDetailStore>
+  @FocusState var focus: Bool
+  var body: some View {
+    VStack(spacing: 13) {
+      TopNavigation(
+        centerTitle: "",
+        trailingItem: (Image.closeCross, {
+          store.send(.didTapDismiss)
+        })
+      )
+      HStack {
+        Text("내용 변경")
+          .notoSans(.headline)
+          .foregroundStyle(Color.greyScale950)
+        Spacer()
+      }
+      .padding(.horizontal, 20)
+      .padding(.vertical, 12)
+      
+      
+      InputField(
+        errorMessage: .constant(nil),
+        text: $store.newTitle,
+        placeHolder: store.selected?.label ?? "내용 변경",
+        isFocused: $focus
+      )
+      
+      
+      VStack {
+        CommonButton(
+          title: "수정완료",
+          style: .default,
+          isActive: store.isActive,
+          action: {store.send(.didTapEditTitleConfirm)}
```

Comment:
```suggestion
          action: { store.send(.didTapEditTitleConfirm) }
```

스페이스 한칸씩 추가 부탁드림니다! 🙇‍♂️